upmed-api/src/api/patient/patient.py

```diff
@@ -1,7 +1,294 @@
-from flask import Blueprint
+from flask import Blueprint, request, jsonify, make_response
+from ....src.util.firebase.db import Database
+from ....src.util.util import Auth
+from ....src.models.patient import Patient
+from ....src.models.health_event import HealthEvent
+
 
 patient_endpoints = Blueprint('patient', __name__)
+pdb = Database()
+pat = pdb.getPatients()
+auth = Auth()
+
 
-@patient_endpoints.route('/')
+@patient_endpoints.route('/', methods=['POST'])
 def root():
-    return "patient"
+    return "GO TO AN ENDPOINT"
+
+
+@patient_endpoints.route('/logIn', methods=['POST'])
+def login():
+    try:
+        pid = request.json['id']
+        email = request.json['email']
+        res = pat.document(str(pid)).get()
+        res = res.to_dict()
+        # print(pid)
+        if res['email'] == email:
+            utype = "PATIENT"
+            auth_token = auth.encode_auth_token(pid, utype)
+            resp = {
+                "googleId": pid,
+                "token": auth_token.decode()
```

Comment:
token should just be the encoded auth_token, don't decode it

Reply:
That is to decode from the byte encoded string to a normal string

Reply:
we want the encoded string to pass around, front end will decode it itself